src/doc/reference.md

```diff
@@ -3603,6 +3603,41 @@ fn main() {
 In this example, the trait `Printable` occurs as a trait object in both the
 type signature of `print`, and the cast expression in `main`.
 
+Trait objects may contain references, and so those references will need a
+lifetime. If the trait contains some sort of lifetime bound, like this:
+
+```rust,ignore
+// Some trait like this...
+trait Bar<'a>: 'a { }
+
+// ... means these are the same:
+Box<Bar>
+Box<Bar + 'a>
+```
+
+If there’s no bound, then the default lifetime is the same as the pointer that
+the trait object is behind:
+
+```rust,ignore
+// Some trait like this...
+trait Bar { }
+
+// ... means these are the same:
+&Bar
+&'a (Bar + 'a)
+```
+
+In any other case, the default is `'static`:
+
+```rust,ignore
+// Some trait like this...
+trait Bar { }
+
+// ... means these are the same:
+Box<Bar>
+Box<Bar + 'static>
```

Comment:
there is technically I think one other rule, not sure how much detail we want to go into though. I think if you have `struct Ref<'a, T: ?Sized>` and you do `Ref<'a, Trait>` you get `Ref<'a, Trait+'a>` by default. Unless we changed that in the follow-up RFC. :P I guess I should go double check.